staramr/detection/AMRDetection.py

```diff
@@ -96,6 +97,13 @@ def _create_plasmidfinder_dataframe(self, plasmidfinder_blast_map: Dict[str, Bla
                                                                genes_to_exclude=self._genes_to_exclude)
         return plasmidfinder_parser.parse_results()
 
+    def _create_quality_module_dataframe(self,files,genome_size_lower_bound,genome_size_upper_bound,minimum_N50_value,
+                                        minimum_contig_length,unacceptable_num_contigs) ->DataFrame:
+        quality_module = QualityModule(files,genome_size_lower_bound,genome_size_upper_bound,minimum_N50_value,
+                                        minimum_contig_length,unacceptable_num_contigs)
+
+        return quality_module._create_quality_module_dataframe()
```

Comment:
This is mostly a notation comment, but the underscore (`_`) in front of a method name is meant to indicate that the method is private (i.e., it is not directly called outside of the class). Since `_create_quality_module_dataframe()` is being called outside of the defining class it is likely not meant to be private. So, could you rename it to `create_quality_module_dataframe()`?